src/dlgAboutDialog.cpp

```diff
@@ -1077,13 +1077,25 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
         // clang-format on
     }
 
-    QString supporters_text(qsl(R"(
-               <p align="center"><br>%1<br></p>
-               %2
-               )")
-                  .arg(tr(R"(
-                          These formidable folks will be fondly remembered forever<br>for their generous financial support on <a href="https://www.patreon.com/mudlet">Mudlet's patreon</a>:
-                          )"), supporters_image_html));
+    QSettings settings;
+    QString supporters_text;
+    if (settings.value("steamMode").toBool()) {
+        supporters_text = qsl(R"(
+                <p align="center"><br>%1<br></p>
+                %2
+                )")
+                    .arg(tr(R"(
+                            These formidable folks will be fondly remembered forever<br>for their generous financial support:
```

Comment:
We don't have to hide that we have a place to accept donations do we - we just must not have an active link to it - right?
```suggestion
                            These formidable folks will be fondly remembered forever<br>for their generous financial support on Mudlet's patreon:
```

Reply:
My interpretation of the comment from Steam when the build was rejected can be summarized as:
> You may thank donators and patrons but call-to-action links should be removed from the game.

The change here attempts to meet those requirements while still retaining the recognition of those who so generously supported Mudlet's development financially.

Reply:
What I am suggesting is keep the displayed text the same (well once the translators have processed it :grin:) just remove the hyperlink itself - so there is nothing to click on...

Reply:
I'm not sure if Steam is okay with that or not, it _might_ be. Many platforms (regardless whether game or app) usually prefer the only mention of donations be done with an associated method of donating via their platform: Apple's App Store and Google Play Store both have in-app purchases, Discord has Server Shops, etc.

Reply:
> ... Many platforms (regardless whether game or app) usually prefer the only mention of donations be done with an associated method of donating via their platform: Apple's App Store and Google Play Store both have in-app purchases, Discord has Server Shops, etc. ...

Usually isn't that so they can take their cut? 🙁 💸